dev/integration_tests/flutter_gallery/linux/flutter/generated_plugins.cmake

```diff
@@ -6,6 +6,9 @@ list(APPEND FLUTTER_PLUGIN_LIST
   url_launcher_linux
 )
 
+list(APPEND FLUTTER_FFI_PLUGIN_LIST
```

Comment:
Is the change to this file related to this PR?

Reply:
This probably comes from running pub get aftr the FFI plugin change.

Reply:
This gets auto updated whenever I run it, and it looks like its not gitignored....

Should be safe to check in :D